ferrocene/doc/release-notes/src/24.05.0.rst

```diff
@@ -21,6 +21,11 @@ New features
   for safety critical use, in addition to being used as a host platform like in
   previous releases.
 
+* Host targets now distribute all the official Rust dev tools. Additionally to
```

Comment:
This should be under experimental features.

Reply:
Done